test/fixtures/cortege_lineups.yml

```diff
@@ -0,0 +1,22 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
If you're not using the fixture in a test, consider removing the fixture completely.